test/listboughtalbums.js

```diff
@@ -0,0 +1,29 @@
+const { authenticateUser, buyAlbum } = require('./utils');
+const { insertUser } = require('./factories');
+
+describe('GET /users/:id/albums', () => {
+  test('should list bought albums', async () => {
+    const user = await insertUser();
+    const loginResponse = await authenticateUser({ ...user, password: '123123123' });
+    const token = `Bearer ${loginResponse.headers.authorization.substring(8)}`;
+    const response = await buyAlbum(1, token);
+    expect(response.statusCode).toBe(201);
+  });
+
+  test('should list bought albums', async () => {
+    const user = await insertUser();
+    const loginResponse = await authenticateUser({ ...user, password: '123123123' });
+    const token = `Bearer ${loginResponse.headers.authorization.substring(8)}`;
+    const response = await buyAlbum(1, token);
+    expect(response.statusCode).toBe(201);
+  });
+
+  test('should not list bought albums', async () => {
+    const user = await insertUser();
+    const loginResponse = await authenticateUser({ ...user, password: '123123123' });
+    const token = `Bearer ${loginResponse.headers.authorization.substring(8)}`;
```

Comment:
Move this to a helper function: `loginResponse.headers.authorization.substring(8)`